### PR TITLE
update folder structure tabs

### LINKED ISSUE #108: Issue on page /project_structure.html
No new lines in project structure R/python examples - just looks like lists of file names

### PR DESCRIPTION
**Pull request by RuSnook**
Added double space to force new lines and added extra dashes to force indent as couldn't find an easy way to make blank space.

This solves #108

### CODE REVIEW

**Comment by RuSnook** [comment]
Just spotted that we also have the same problem on the testing code page too.
Will update that too.

**Comment by RuSnook** [comment]
now corrected the problem properly rather than the other fix I found. hadn't realised that the bars were not the correct type.

File: book/project_structure.md [hunk 1]
```diff
@@ -80,45 +80,73 @@ Most analysis will have an ingest or input stage, a processing stage, and a repo
...
+│   │   ├── **/*.css
+│   ├── images
+│   ├── js
+│   ├── index.html
+├── dist (or build
+├── node_modules
+├── package.json
+├── package-lock.json 
+└── .gitignore
+```
```
Comment (rowanhemsi) [inline book/project_structure.md]: This is not needed

**Review by rowanhemsi — APPROVED** [review]
The tab sections look good, just need to remove the extra example indicated

**Pull request merged**